mathesar_ui/src/api/rpc/databases.ts

```diff
@@ -2,17 +2,100 @@ import { rpcMethodTypeContainer } from '@mathesar/packages/json-rpc-client-build
 
 import type { Server } from './servers';
 
-export interface Database {
+export interface DatabaseResponse {
   id: number;
   name: string;
   server_id: Server['id'];
 }
 
+/**
+ * TODO: Figure out a better place to move classes like these.
```

Comment:
I like this idea, but I have some different opinions regarding the rules. It would be nice to chat about this on a call.

Also, it's nice seeing this sort of proposal written down, but process-wise I don't like using code comments as a discussion channel. I'd prefer to use a wiki PR or github issue.

Reply:
We could resolve this in the proposal PRs and on call. I've removed the discussion code comments.